excalidraw-app/data/LocalData.ts

```diff
@@ -183,3 +198,55 @@ export class LocalData {
     },
   });
 }
+
+class LibraryIndexedDBAdapter implements LibraryPersistenceAdapter {
```

Comment:
we may wanna abstract export this from excal package (we'd have to install `idb-keyval` into core editor unless we made it optional peer dep)

/cc @ad1992